docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+release notes from the 8.0.0-alpha1, -alpha2, -beta1, -rc1 and -rc2 releases.
+
+Also see <<breaking-changes-8.0,Breaking changes in 8.0>>.
+
+[[breaking-8.0.0]]
+[float]
+=== Breaking changes
+
+Aggregations::
+* Percentiles aggregation: disallow specifying same percentile values twice {es-pull}52257[#52257] (issue: {es-issue}51871[#51871])
+* Remove Adjacency_matrix setting {es-pull}46327[#46327] (issues: {es-issue}46257[#46257], {es-issue}46324[#46324])
+* Remove `MovingAverage` pipeline aggregation {es-pull}39328[#39328]
+* Remove deprecated `_time` and `_term` sort orders {es-pull}39450[#39450]
+* Remove deprecated date histo interval {es-pull}75000[#75000]
+
+Allocation::
+* Breaking change for single data node setting {es-pull}73737[#73737] (issues: {es-issue}55805[#55805], {es-issue}73733[#73733])
+* Remove `include_relocations` setting {es-pull}47717[#47717] (issues: {es-issue}46079[#46079], {es-issue}47443[#47443])
+
+Analysis::
+* Cleanup versioned deprecations in analysis {es-pull}41560[#41560] (issue: {es-issue}41164[#41164])
+* Remove preconfigured `delimited_payload_filter` {es-pull}43686[#43686] (issues: {es-issue}41560[#41560], {es-issue}43684[#43684])
+
+Authentication::
+* Always add file and native realms unless explicitly disabled {es-pull}69096[#69096] (issue: {es-issue}50892[#50892])
+* Do not set a NameID format in Policy by default {es-pull}44090[#44090] (issue: {es-issue}40353[#40353])
+* Make order setting mandatory for Realm config {es-pull}51195[#51195] (issue: {es-issue}37614[#37614])
+
+CCR::
+* Avoid auto following leader system indices in CCR {es-pull}72815[#72815] (issue: {es-issue}67686[#67686])
+
+Cluster Coordination::
+* Remove join timeout {es-pull}60873[#60873] (issue: {es-issue}60872[#60872])
+* Remove node filters for voting config exclusions {es-pull}55673[#55673] (issues: {es-issue}47990[#47990], {es-issue}50836[#50836])
+* Remove support for delaying state recovery pending master {es-pull}53845[#53845] (issue: {es-issue}51806[#51806])
+
+Distributed::
+* Remove synced flush {es-pull}50882[#50882] (issues: {es-issue}50776[#50776], {es-issue}50835[#50835])
+* Remove the `cluster.remote.connect` setting {es-pull}54175[#54175] (issue: {es-issue}53924[#53924])
+
+Engine::
+* Force merge should reject requests with `only_expunge_deletes` and `max_num_segments` set {es-pull}44761[#44761] (issue: {es-issue}43102[#43102])
+* Remove per-type indexing stats {es-pull}47203[#47203] (issue: {es-issue}41059[#41059])
+* Remove translog retention settings {es-pull}51697[#51697] (issue: {es-issue}50775[#50775])
+
+Features/CAT APIs::
+* Remove the deprecated `local` parameter for `_cat/indices` {es-pull}64868[#64868] (issue: {es-issue}62198[#62198])
+* Remove the deprecated `local` parameter for `_cat/shards` {es-pull}64867[#64867] (issue: {es-issue}62197[#62197])
+
+Features/Features::
+* Remove deprecated ._tier allocation filtering settings {es-pull}73074[#73074] (issue: {es-issue}72835[#72835])
```

Comment:
Nit:
```suggestion
* Remove deprecated `._tier` allocation filtering settings {es-pull}73074[#73074] (issue: {es-issue}72835[#72835])
```

Reply:
Fixed in https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c